components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/addapi-ajaxprocessor.jsp

```diff
@@ -49,6 +49,13 @@
 	String apiContext = request.getParameter("apiContext");
 	String hostname = request.getParameter("hostname");
 	String port = request.getParameter("port");
+	String version = request.getParameter("version");
+	String versionType = request.getParameter("versionType");
+
+	if ("none".equals(versionType)) {
+	    // in synapse, default version type is picked from empty string
+	    versionType = "";
+    }
 	
 	if(port == null || "".equals(port)){

```

Comment:
Can we use isEmpty() here ?

Reply:
The reason i did not change these places because these were there in the original class file and if we are to do the change, then there will be about hundreds of places which have to be changed. Then there will be unnecessary changes in this PR. So what should we do?